src/encoder/encoderfdkaac.cpp

```diff
@@ -0,0 +1,408 @@
+// encoderfdkaac.cpp
+// Created on Aug 15 2017 by Palakis
+
+#include <QDesktopServices>
+#include <QDir>
+#include <QString>
+#include <QStringList>
+
+#include "recording/defs_recording.h"
+#include "util/logger.h"
+#include "util/sample.h"
+#include "engine/sidechain/enginesidechain.h"
+
+#include "encoder/encoderfdkaac.h"
+
+namespace {
+// recommended in encoder documentation, section 2.4.1
+const int kOutBufferBits = 6144;
+const mixxx::Logger kLogger("EncoderFdkAac");
+}
+
+EncoderFdkAac::EncoderFdkAac(EncoderCallback* pCallback, QString pFormat)
+    : aacEncOpen(nullptr),
+      aacEncClose(nullptr),
+      aacEncEncode(nullptr),
+      aacEncInfo(nullptr),
+      aacEncoder_SetParam(nullptr),
+      m_aacAot(0),
+      m_bitrate(0),
+      m_channels(0),
+      m_samplerate(0),
+      m_pCallback(pCallback),
+      m_library(nullptr),
+      m_pInputFifo(nullptr),
+      m_pFifoChunkBuffer(nullptr),
+      m_readRequired(0),
+      m_aacEnc(),
```

Comment:
I don't think it will be initialized unless it's explicitly in this list, will it?

https://godbolt.org/z/2VaJx5
vs.
https://godbolt.org/z/Dn8zTE

Reply:
Thank you. Interesting, I did not know about this.
This is actually true for pod types.
: m_pod() 
is the same as 
: m_pod(0)

I was also wrong that HANDLE_AACENCODER is a struct. It is  
AACENCODER*

So I think in this case we should go for 
m_aacEnc(nullptr),
to avoid future confusion.